Sources/PromotedCore/NetworkConnection.swift

```diff
@@ -85,18 +85,26 @@ public extension NetworkConnection {
     if !apiKey.isEmpty {
       request.addValue(apiKey, forHTTPHeaderField: "x-api-key")
     }
-    if (
-      clientConfig.metricsLoggingWireFormat == .binary &&
-      clientConfig.metricsLoggingURL.contains(".promoted.ai") &&
-      request.value(forHTTPHeaderField: "content-type") == nil
-    ) {
+    if (request.value(forHTTPHeaderField: "content-type") == nil) {
+      let contentTypeValue = contentType(clientConfig: clientConfig)
       request.addValue(
-        "application/protobuf",
+        contentTypeValue,
         forHTTPHeaderField: "content-type"
       )
     }
     return request
   }
+
+  func contentType(
+    clientConfig: ClientConfig
+  ) -> String {
+    switch clientConfig.metricsLoggingWireFormat {
+    case .binary:
+      return "application/protobuf"
+    case .json:
+      return "application/json"
```

Comment:
Didn't have too much context on this, but I assume JSON and proto binary are the only supported types here.

Reply:
Yes.